backend/packages/Upgrade/src/api/controllers/ExperimentExcludeController.ts

```diff
@@ -0,0 +1,339 @@
+          JSON.stringify({ type: SERVER_ERROR.INCORRECT_PARAM_FORMAT, message: ' : id should be of type UUID.' })
+        )
+      );
+    }
+    return this.experimentExclude.deleteExperimentUser(userId, experimentId);
+  }
+
+  /**
+   * @swagger
+   * /experimentExclude/group:
+   *    get:
+   *       description: Get all Excluded Groups for an experiment
+   *       tags:
+   *         - Exclude
+   *       produces:
+   *         - application/json
+   *       responses:
+   *          '200':
+   *            description: All Excluded Groups for an experiment
+   *            schema:
+   *              $ref: '#/definitions/userExperimentExcludeResponse'
+   */
+  @Get('/group')
+  public getExperimentExcludedGroups(): Promise<ExplicitExperimentGroupExclusion[]> {
+    return this.experimentExclude.getAllExperimentGroups();
+  }
+
+  @Get('/group/:type/:id/:experimentId')
+  public getExperimentExcludedGroupById(
+    @Param('type') type: string,
+    @Param('id') groupId: string,
+    @Param('experimentId') experimentId: string
+    ): Promise<ExplicitExperimentGroupExclusion> {
+    if (!type) {
+      return Promise.reject(new Error(SERVER_ERROR.MISSING_PARAMS + ' : type should not be null.'));
+    }
+    if (!groupId) {
+      return Promise.reject(new Error(SERVER_ERROR.MISSING_PARAMS + ' : groupId should not be null.'));
+    }
+    if (!experimentId) {
+      return Promise.reject(new Error(SERVER_ERROR.MISSING_PARAMS + ' : experiment should not be null.'));
+    }
+    if (!validator.isUUID(experimentId)) {
+      return Promise.reject(
+        new Error(
+          JSON.stringify({ type: SERVER_ERROR.INCORRECT_PARAM_FORMAT, message: ' : id should be of type UUID.' })
+        )
+      );
+    }
+    return this.experimentExclude.getExperimentGroupById(type, groupId, experimentId);
+  }
+
+  /**
+   * @swagger
+   * /experimentExclude/group:
+   *    put:
+   *       description: Exclude a Group from an experiment
+   *       consumes:
+   *         - application/json
+   *       parameters:
+   *         - in: body
+   *           name: body
+   *           required: true
+   *           schema:
+   *             type: object
+   *             required:
+   *               - id
+   *               - type
+   *             properties:
+   *               id:
+   *                type: string
+   *               type:
+   *                type: string
+   *       tags:
+   *         - ExperimentExclude
+   *       produces:
+   *         - application/json
+   *       responses:
+   *          '200':
+   *            description: Exclude group from an experiment
+   *            schema:
+   *              $ref: '#/definitions/userExperimentExcludeResponse'
+   */
+  @Post('/group')
+  public experimentExcludeGroup(
+    @BodyParam('groups') groups: Array<{ groupId: string, type: string }>,
+    @BodyParam('experimentId') experimentId: string    
+    ): Promise<ExplicitExperimentGroupExclusion[]> {
+      console.log(' point 1');
```

Comment:
Remove this

Reply:
@YashilDepani Remove the console logs if any other as well.

Reply:
@RidhamShah can you provide update on this?